tests/e2e/test_vdbench_workload.py

```diff
@@ -0,0 +1,279 @@
+"""
+Test to exercise VDBench Workload
+"""
+import logging
+import pytest
+import time
+import os
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import TimeoutSampler, ocsci_log_path
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, workloads
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+# TODO: move this function to pod.py helper functions
+def download(pod_name, localpath, remotepath):
```

Comment:
`download` seems to be ambiguous. pls consider a more meaningful name. 

p.s., I see there is already a function `upload`, maybe you used the same terminology. But, it is better to use a mre meaningful name.

Reply:
changed.